src/integrationTest/sam.test.ts

```diff
@@ -343,10 +331,33 @@ describe('SAM Integration Tests', async () => {
                         )
                     })
 
-                    await vscode.commands.executeCommand<any>(
-                        codeLens.command!.command,
-                        ...codeLens.command!.arguments!
-                    )
+                    const rootUri = vscode.Uri.file(appPath)
+                    const launchConfig = new LaunchConfiguration(rootUri)
+                    const testConfig = {
+                        type: 'aws-sam',
+                        request: 'direct-invoke',
+                        name: 'test-config-1',
+                        invokeTarget: {
+                            target: 'template',
+                            logicalId: 'HelloWorldFunction',
+                            templatePath: cfnTemplatePath,
+                            //projectRoot: subSuiteTestLocation,
+                            //lambdaHandler: 'StockBuyer::StockBuyer.Function::FunctionHandler',
+                        },
+                        lambda: {
+                            environmentVariables: {},
+                            payload: {},
+                            // runtime: scenario.runtime,
+                        },
+                    }
+                    // TODO: launchConfig.getDebugConfigurations() is empty after this,
+                    // but launch.json *does* have the content in it, so F5 works.
+                    // Bug/quirk with LaunchConfiguration impl?
+                    await launchConfig.addDebugConfiguration(testConfig)
+
+                    // Invoke "F5".
+                    await vscode.commands.executeCommand('workbench.action.debug.start')
```

Comment:
How do we know that the `testConfig` entry is being invoked?

Reply:
because there's only 1 launch config. and because the `scenario` properties match those in the test. 

however i will also change the test to clear out the launch configs first.

Reply:
Try `vscode.debug.startDebugging` instead? https://code.visualstudio.com/api/references/vscode-api#debug

Reply:
why? hitting F5 is what an actual user would do.

Reply:
Could this potentially lead to flakiness though? Something like "user selects the debug config and presses F5" seems like more of a UI test, and honestly not one that we should be responsible for (unless we ever add a debug config that has a stipulation like "Editor of type X also has to be open before pressing F5".